hackclub.com.yaml

```
@@ -2068,6 +2072,10 @@ newcanaanlib:
   ttl: 600
   type: CNAME
   value: cname.vercel-dns.com.
+neko:
+ttl: 600
+type: CNAME
+value: cat-jammers.github.io.
```

Can you fix the indents here?
```suggestion
neko:
  ttl: 600
  type: CNAME
  value: cat-jammers.github.io.
```